letsencrypt/plugins/standalone.py

```diff
@@ -182,7 +176,11 @@ def supported_challenges(self):
                    self.conf("supported-challenges").split(","))
 
     def more_info(self):  # pylint: disable=missing-docstring
-        return self.__doc__
+        return """\
+This authenticator creates its own ephemeral TCP listener on the
+ necessary port in order to respond to incoming DVSNI and SimpleHTTP
+ challenges from the certificate authority. Therefore, it does not
+ rely on any existing server program.""".replace("\n", "")
```

Comment:
Maybe `replace("\n", " ")` and get rid of the 1 char indentation throughout the text? This will make text more readable and is easier not to accidentally mess up.